include/zephyr/drivers/clock_control/stm32_clock_control.h

```diff
@@ -281,27 +281,19 @@ struct stm32_pclken {
 
 /** Device tree clocks helpers  */
 
-#define STM32_CLOCK_INFO(clk_index, id)					\
+#define STM32_CLOCK_INFO(clk_index, node_id)				\
```

Comment:
btw, it would be nice to align inst/non-inst, e.g. `STM32_DT_CLOCKS()` and `STM32_DT_INST_CLOCKS()`